platform/web/web_main.cpp

```diff
@@ -140,5 +138,5 @@ extern EMSCRIPTEN_KEEPALIVE int godot_web_main(int argc, char *argv[]) {
 	// We are inside an animation frame, we want to immediately draw on the newly setup canvas.
 	main_loop_callback();
 
-	return 0;
+	return os->get_exit_code();
```

Comment:
CC @adamscott @Faless - I changed this for consistency with other platforms, but I don't know if this can have unwanted implications.